src/items.js

```diff
@@ -0,0 +1,137 @@
+    update(tasks);
+  } else if (tasks[index].completed === false) {
+    tasks[index].completed = true;
+    update(tasks);
+  }
+}
+
+function addTodo(e) {
+  e.preventDefault();
+  // Create list
+  creatitem(input.value);
+  const newTodo = document.createElement('li');
+  newTodo.innerText = input.value;
+  newTodo.className = 'list-item';
+
+  list.appendChild(newTodo);
+  input.value = '';
+  // Create Completed Button
+  const checkbox = document.createElement('INPUT');
+  checkbox.setAttribute('type', 'checkbox');
+  checkbox.id = 'checkbox';
+  newTodo.appendChild(checkbox);
+
+  checkbox.addEventListener('change', () => {
+    changeStatus(checkbox, tasks.length - 1);
+  });
+
+  // // Create trash button
+  const trashButton = document.createElement('button');
+  trashButton.innerHTML = 'DELET';
+  trashButton.className = 'trash-btn';
+  newTodo.appendChild(trashButton);
+
+  // // Create edit button
+  const editButton = document.createElement('button');
+  editButton.innerHTML = 'edit';
+  editButton.className = 'edit-btn';
+  newTodo.appendChild(editButton);
+}
+
+function newTask() {
+  list.innerHTML = '';
+  tasks.forEach((item, i) => {
+    const li = document.createElement('li');
+    li.className = 'list-item';
+    li.innerText = tasks[i].description;
+    li.id = i;
+    list.appendChild(li);
+    const checkbox = document.createElement('INPUT');
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.checked = tasks[i].completed;
+    checkbox.id = 'checkbox';
+    li.appendChild(checkbox);
+    checkbox.addEventListener('change', () => {
+      changeStatus(checkbox, i);
+    });
+    const trashButton = document.createElement('button');
+    trashButton.innerText = 'DELET';
+    trashButton.className = 'trash-btn';
+    li.appendChild(trashButton);
+
+    // // Create edit button
+    const editButton = document.createElement('button');
+    editButton.innerText = 'edit';
+    editButton.className = 'edit-btn';
+    li.appendChild(editButton);
+
+    editButton.addEventListener('click', (event) => {
+      const newTask = prompt('enter');
+      const newId = event.target.parentNode.getAttribute('id');
+      tasks[newId].description = newTask;
+      update(tasks);
+      const anotherT = document.querySelector(`li[id="${newId}"]`);
+      anotherT.innerHTML = newTask;
+    });
+  });
+}
+
+function removeLocalTodos(tasks) {
+  if (localStorage.getItem('tasks') === null) {
+    // local
+  } else {
+    tasks = JSON.parse(localStorage.getItem('tasks'));
+  }
+  const taskIndex = tasks.children;
+  tasks.splice(tasks.indexOf(taskIndex), 1);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+function deleteitem(e) {
+  const item = e.target;
+  if (item.classList[0] === 'trash-btn') {
+    const todo = item.parentElement;
+    todo.remove();
+    removeLocalTodos(todo);
+  }
+}
+
+list.addEventListener('click', deleteitem);
+
+const clearAll = document.querySelector('.clear');
+
+function clear() {
```

Comment:
- [ ] Here on clear is a little diff as expected, when I have some elements, for example:
![image](https://user-images.githubusercontent.com/20563252/134753135-ad6b5020-33ac-4817-a694-61954cb285f8.png)
- Then, I delete:
![image](https://user-images.githubusercontent.com/20563252/134753160-6d653a52-3e08-4747-a735-3635317dab61.png)
- This is the result if I refresh the page:
![image](https://user-images.githubusercontent.com/20563252/134753172-b09d366e-19e9-46c6-b260-0cd46e171cfe.png)
- I think this problem is about the id's issue, please, check this bug and if the id is good, maybe this issue is solved too. 🐱